server/controllers/unfurl.js

```diff
@@ -340,8 +340,8 @@ const getInsightAttachment = (insight, owner, params) => {
       }
     ]
   };
-  if (insight.body.imageUrl) {
-    attachment.imageUrl = insight.body.imageUrl;
+  if (!attachment.image_url) {
```

Comment:
What happens if insight.body.imageUrl is undefined?

Reply:
@rflprr There will be no image in the attachment displayed in slack.